### PR TITLE
support-full-scan-in-match-&multitag-condition

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 1]
```diff
@@ -1,107 +1,44 @@
 # MATCH
 
-`MATCH`语句提供基于模式（pattern）匹配的搜索功能。
+`MATCH`语句提供基于模式（Pattern）匹配的搜索功能，其通过定义一个或多个搜索模式，允许在 NebulaGraph 中查找与模式匹配的数据。在检索到匹配的数据后，用户可以使用 RETURN 子句将其作为结果返回。
```
Comment (cooper-lzy) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:3]:
```suggestion
`MATCH`语句提供基于模式（Pattern）匹配的搜索功能，其通过定义一个或多个模式，允许在 NebulaGraph 中查找与模式匹配的数据。在检索到匹配的数据后，用户可以使用 RETURN 子句将其作为结果返回。
```

**Review by cooper-lzy — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 2]
```diff
@@ -1,107 +1,46 @@
...
 !!! compatibility "历史版本兼容性" 
 
-    从 3.0.0 版本开始，为了区别不同 Tag 的属性，返回属性时必须额外指定 Tag 名称。即从`RETURN <变量名>.<属性名>`改为`RETURN <变量名>.<Tag名>.<属性名>`。
+    - 从 3.5.0 版本开始，`MATCH`语句支持全表扫描，即在不使用任何索引或者过滤条件的情况下可遍历图中点或边。在此之前的版本中，MATCH 语句在某些情况下需要索引才能执行查询或者需要使用`LIMIT`限制输出结果数量。
+    - 从 3.0.0 版本开始，为了区别不同 Tag 的属性，返回属性时必须额外指定 Tag 名称。即从`RETURN <变量名>.<属性名>`改为`RETURN <变量名>.<Tag名>.<属性名>`。
 
 
+## 使用说明
 
-## 示例
```
Comment (Shylock-Hg) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:50]: Why delete these?

File: docs-2.0/3.ngql-guide/14.native-index-statements/README.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+
 NebulaGraph 支持两种类型索引：原生索引和全文索引。
 
-和一般数据库意义上的索引概念不同，NebulaGraph 中的索引没有加速查询的功能，是用于定位到数据的必要前置条件。
+## 使用说明
+
+- 索引可以提高查询性能，但是会降低写入性能。
+- 索引是执行`LOOKUP`语句时用于定位到数据的前置条件，如果没有索引，执行`LOOKUP`语句会报错。
+- 使用索引时，NebulaGraph 会自动选择最优的索引。
+- 具有高选择度的索引（即属性的值的种类较少，例如身份账号）可显著提升查询性能；而对于低选择度的索引（即属性的值的种类较多，例如国家），查询性能可能不会带来显著提升。
```
Comment (cooper-lzy) [inline docs-2.0/3.ngql-guide/14.native-index-statements/README.md:12]:
```suggestion
- 具有高选择度的索引（即索引列中不同值的记录数与总记录数的比值较高，例如身份证的比值为 1）可显著提升查询性能；而对于低选择度的索引（例如国家），查询性能可能不会带来显著提升。
```

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md (hunk 2 above)
Comment (abby-cyber) [reply docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:50]: > Why delete these?

 just changed this name to `use patterns`

**Review by cooper-lzy — DISMISSED** [review]

File: docs-2.0/2.quick-start/6.cheatsheet-for-ngql-command.md [hunk 4]
```diff
@@ -327,7 +327,7 @@
...
 | [SAMPLE](../3.ngql-guide/8.clauses-and-options/sample.md) | `<go_statement> SAMPLE <sample_list>;`         | `GO 3 STEPS FROM "player100"  OVER *   YIELD properties($$).name AS NAME, properties($$).age AS Age  SAMPLE [1,2,3];` | 在结果集中均匀取样并返回指定数量的数据。 |
 | [ORDER BY](../3.ngql-guide/8.clauses-and-options/order-by.md) | `<YIELD clause> ORDER BY <expression> [ASC | DESC] [, <expression> [ASC | DESC] ...]` | `FETCH PROP ON player "player100", "player101", "player102", "player103" YIELD player.age AS age, player.name AS name | ORDER BY $-.age ASC, $-.name DESC` | `ORDER BY`子句指定输出结果的排序规则。                       |
-| [RETURN](../3.ngql-guide/8.clauses-and-options/return.md) | `RETURN {<vertex_name>|<edge_name>|<vertex_name>.<property>|<edge_name>.<property>|...}` | `MATCH (v:player) RETURN v.player.name, v.player.age LIMIT 3`              | 返回点的属性为`name`和`age`的前三行值。                            |
+| [RETURN](../3.ngql-guide/8.clauses-and-options/return.md) | `RETURN {<vertex_name>|<edge_name>|<vertex_name>.<property>|<edge_name>.<property>|...}` | `MATCH (v:player) RETURN v.player.name, v.player.age`              | 返回点的属性为`name`和`age`的前三行值。                            |
```
[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/2.quick-start/6.cheatsheet-for-ngql-command.md]: > "前三行值。" is not valid :P

fixed

**Review by cooper-lzy — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**